FetchMigration/README.md

```diff
@@ -67,42 +60,46 @@ docker build -t fetch-migration .
 ```
 
 Then run the `fetch-migration` image.
-Replace `<pipeline_yaml_path>` in the command below with the path to your Logstash config file:
+Replace `<pipeline_yaml_path>` in the command below with the path to your Data Prepper pipeline `yaml` file:
 
 ```shell
-docker run -p 4900:4900 -v <pipeline_yaml_path>:/code/input.yaml ict
+docker run -p 4900:4900 -v <pipeline_yaml_path>:/code/input.yaml fetch-migration
 ```
 
+### AWS deployment
+
+Refer to [AWS Deployment](../deployment/README.md) to deploy this solution to AWS.
+
 ## Development
 
-The source code for the tool is located under the `index_configuration_tool/` directory. Please refer to the [Setup](#setup) section to ensure that the necessary dependencies are installed prior to development.
+The source code for the tool is located under the `python/` directory, with unit test in the `tests/` subdirectory. 
+Please refer to the [Setup](#setup) section to ensure that the necessary dependencies are installed prior to development.
 
 Additionally, you'll also need to install development dependencies by running:
 
 ```shell
-python -m pip install -r index_configuration_tool/dev-requirements.txt
+python -m pip install -r python/dev-requirements.txt
 ```
 
 ### Unit Tests
 
-Unit tests are located in a sub-directory named `tests`. Unit tests can be run using:
+Unit tests can be run from the `python/` directory using:
 
 ```shell
-python -m unittest
+python -m coverage run -m unittest
 ```
 
-### Coverage
-
-Code coverage metrics can be generated by first running unit tests using _coverage run_:
+_Code coverage_ metrics can be generated after a unit-test run. A report can either be printed on the command line:
```

Comment:
nit: not sure why the ### Coverage (h3) was removed.

Reply:
Good find, that was a mistake. Fixed now.